src/planning/speculator.ts

```diff
@@ -56,6 +56,7 @@ export class Speculator {
         description,
         arc.modality,
         arc.pec.slotComposer ? arc.pec.slotComposer.modalityHandler.descriptionFormatter : undefined);
+
```

Comment:
please, revert this file, there is no changes here.

Reply:
Done.